relay-event-schema/src/protocol/contexts/mod.rs

```diff
@@ -287,17 +287,10 @@ impl FromValue for Contexts {
     }
 }
 
-/// Returns `true` if `value` is `None`, empty object, or an empty array.
-fn value_is_empty(value: Option<&Annotated<Value>>) -> bool {
-    let Some(value) = value.and_then(|v| v.value()) else {
-        return true;
-    };
-
-    match value {
-        Value::Array(values) => values.is_empty(),
-        Value::Object(values) => values.is_empty(),
-        _ => false,
-    }
+/// Returns `true` if `value` is a nonempty string, which is the only valid form
+/// for the `"type"` field of a context.
```

Comment:
```suggestion
/// Returns `true` if `value` is a non-empty string, which is the only valid value
/// for the `"type"` field of a context.
```

Maybe value over form, as emptiness is more value than form.